HowToRun/RunOnDocker.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Executar em Docker
+parent: Como executar o projeto
+nav_order: 3
+---
+
+# Como executar o projeto em Docker
+
+Além das opções previamente exploradas sobre como executar seus projetos com o WSL e com o Ubuntu, agora iremos conhecer uma terceira ferramenta: o Docker. Esta ferramenta consiste basicamente em uma espécie de máquina virtual que contém todos os *softwares* necessários para que você execute seu projeto.
+
+Antes de tudo, verifique se o Docker está abrto. Para isso, abra o aplicativo do Docker no seu computador e veja se a seguinte tela está aparecendo:
+
+![Docker is running](/assets/img/Docker/dockerrunning.png)
+
+Caso esteja desse jeito, é só seguir o tutorial!
+
+## Como configurar o Docker para rodar seu projeto
+
+Primeiramente, é necessário fazer o *download* do *Dockerfile*. Este arquivo será o responsável por conter as dependências do seu projeto. O arquivo pode ser baixado por meio **[deste link](https://github.com/ThundeRatz/DockerCarrinhos)**.
+
+Após fazer o *download* deste arquivo, abra o **prompt de comando**/*command prompt* e redirecione para a pasta que o arquivo está. O modo mais fácil de fazer isso é copiando o caminho do arquivo e digitando no prompt de comando:
```

Comment:
```suggestion
Após fazer o *download* deste arquivo, abra o **prompt de comando**/**command prompt** e redirecione para a pasta que o arquivo está. O modo mais fácil de fazer isso é copiando o caminho do arquivo e digitando no prompt de comando:
```

Reply:
Então, essa parte eu segui o padrão do WSL que estava fazendo partes importantes em negrito e a tradução em ingles delas em itálico